_episodes/01-introduction.md

```diff
@@ -25,8 +27,22 @@ CWL workflows are written in a subset of YAML, with a syntax that does not restr
 
 The CWL standards explicitly support the use of software container technologies, helping ensure that the execution of tools is reproducible. Data locations are explicitly defined, and working directories kept separate for each tool invocation. These standards ensure the portability of tools and workflows, allowing the same workflows to be run on your local machine, or in a HPC or cloud environment, with minimal changes required.
 
+# RNA sequencing example
+
+In this tutorial a bio-informatics RNA-sequencing analysis is used as an example. However, there is no specific knowledge needed for this tutorial. RNA-sequencing is a technique which examines the quantity and sequences of RNA in a sample using next-generation sequencing. The RNA reads are analzed to measure the relative numbers of different RNA molecules in the sample. This analysis is differential gene expression.
+
+The process looks like this:
+
+![](../assets/imp/RNAseqWorkflow.png){: height="400px"}
```

Comment:
see https://carpentries.github.io/lesson-example/03-organization/index.html#figures  and

```suggestion
![]({{page.root}}/fig/RNAseqWorkflow.png){: height="400px"}
```